content/zh/blog/_posts/2019-12-09-kubernetes-1.17-release-announcement.md

```diff
@@ -0,0 +1,274 @@
+---
+layout: blog
+title: "Kubernetes 1.17：稳定"
+date: 2019-12-09T13：00：00-08：00
```

Comment:
@rudy2steiner sorry I just spotted another thing to fix:

```suggestion
date: 2019-12-09T13:00:00-08:00
```

These need to be normal (us-ascii) colons. If you used a machine translation tool, please bear in mind the advice at https://kubernetes.io/docs/contribute/localization/#translating-content about not relying on automation here.